Modix.Bot/Modules/EmojiStatsModule.cs

```diff
@@ -99,6 +99,7 @@ await ReplyAsync(embed: new EmbedBuilder()
             [Summary("The emoji to retrieve information about.")]
                 IEmote emoji)
         {
+            // TODO: Abstract this as IEmoteEntity instead (actually not entirely sure how this would work, as it comes out of the commands module)
```

Comment:
To resolve this TODO, do we just need changes in the `EmoteTypeReader`? If so, could we just handle that in this PR?

https://github.com/discord-csharp/MODiX/blob/master/Modix.Bot/EmoteTypeReader.cs